TFNetworkRecLayer.py

```diff
@@ -725,6 +727,18 @@ def is_prev_step_layer(cls, layer):
       return layer.is_prev_time_frame
     return False
 
+  def get_sub_layer(self, layer_name):
+    """
+    :param str layer_name:  The sub_layer addressed by '/' separated path.
+    :return: The sub_layer addressed in layer_name or None if no sub_layer exists
+    :rtype: LayerBase|None
+    """
+    if '/' in layer_name and isinstance(self.cell, _SubnetworkRecCell):
```

Comment:
Why the extra `"unit/"`? I think this is confusing, and also more complicated than it needs to be.

Reply:
I would not do that. This is purely artificial. Actually I don't like it too much that it is called `"unit"` currently. Maybe we rename that later. But anyway, this logic is kind of independent (or should be independent) how the parameter of the class is named. So I would just remove that.

Reply:
I can see why this would be simpler (although I still think people will first try to access it through the path they see in the network dictionary).
So I would change this to remain independent of changes in naming of other layers. 
But what would you think of having something like 
```
if 'unit' == layer_name.split('.')[0]: 
  layer_name = layer_name.split('.')[1:] 
```

Reply:
Hm, I don't think so. I would just remove the `"unit"` part. It does not really add any new functionality or ability. And also keep it simple for now, so no extra cases.

Reply:
Ok... I changed it to expect "base_layer/sub_layer"